sympy/printing/pretty/tests/test_pretty.py

```diff
@@ -332,6 +332,11 @@ def test_upretty_modifiers():
     assert upretty( Symbol('x__dot') ) == u('x__dot')
 
 
+def test_pretty_Cycle():
+    from sympy.combinatorics.permutations import Cycle
+    assert pretty(Cycle(1, 2)) == '([1, 2],)'
```

Comment:
I would like that, too. But perhaps even the 'plain' print should be changed. In currently works as follows, with an extra 'Cycle':
```
>>> Cycle(1, 2)
Cycle(1, 2)
>>> Cycle(1, 2)(2, 3)
Cycle(1, 3, 2)
```